exercises/bob/package.yaml

```diff
@@ -3,6 +3,7 @@ version: 1.2.0.5
 
 dependencies:
   - base
+  - safe
```

Comment:
`safe` should not be included in the list of dependencies distributed to students, since not all student solutions will use `safe`